app/models/book_index_state.rb

```diff
@@ -81,6 +83,12 @@ def mark_created
     save!
   end
 
+  def mark_as_http_error
+    self.state = STATE_HTTP_ERROR
+    self.status_log << StatusLog.new(action: StatusLog::ACTION_ERROR)
```

Comment:
when we look in dynamo we won't see that it was an http error (will say "index_error").  should it say "http_error"?  or add "http_error" and then add "index_error"?